packages/zk-certificates/scripts/decryptZkCert.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright (C) 2024 Galactica Network. This file is part of zkKYC. zkKYC is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. zkKYC is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>. */
+
+import { decryptSafely, getEncryptionPublicKey } from '@metamask/eth-sig-util';
+
+import zkCert from '../../../test/encryptedZkCert.json';
+import { testHolder } from '../../snap/test/constants.mock';
+// This is the zkCert you want to decrypt
+
+/**
+ * Script for testing and debugging encrypted ZkCertificates.
+ *
+ * Instructions:
+ * 1. Choose an ecdsa key pair for testing. You can use the one suggested by running this script.
```

Comment:
So it's just an ECDSA key pair! This is the main thing for me: I couldn't find this information anywhere. Please, let's add it to the documentation somewhere. Or just point me where it mentioned

Reply:
Added details in a gitbook PR and requested your review. https://app.gitbook.com/o/IbmhhVJSM8rZ0aECe2R3/s/NMoORBGBxztthVlosoIF/~/diff/~/changes/55/guardian-guide/create-and-issue-zkkyc

Reply:
Approved